### ISSUE TITLE
MA0183: False positive when using extra parentheses

### ISSUE THREAD

**Issue by drieseng** [meziantou/Meziantou.Analyzer#1043]
### Version of the Meziantou.Analyzer NuGet package

3.0.15

### Rule Identifier

MA0183

### Target Framework

.NET 10.0

### C# Language version

C# 14

### Description

When the "placeholder" argument to a formatting method is enclosed in extra parentheses, then MA0183 is wrongfully reported.

### Reproduction Steps

Minimal code:

```c#
using System.Globalization;

_ = string.Format(CultureInfo.InvariantCulture, SR.PlaceHolder, (obj is null ? string.Empty : "X")); // NOK, MA0183 is reported
_ = string.Format(CultureInfo.InvariantCulture, SR.PlaceHolder, obj is null ? string.Empty : "X"); // // OK, MA0183 is not reported

_ = string.Format(CultureInfo.InvariantCulture, SR.NoPlaceHolder, (obj is null ? string.Empty : "X")); // OK, MA0183 is not reported
_ = string.Format(CultureInfo.InvariantCulture, SR.NoPlaceHolder, obj is null ? string.Empty : "X"); // OK, MA0183 is not reported

internal static class SR
{
    public const string PlaceHolder = "Format string with placeholder: '{0}'.";
    public const string NoPlaceHolder = "Format string without placeholder.";
}
```
